dora/core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -1238,6 +1243,46 @@ protected boolean parentExists(String path) throws IOException {
     return parentKey != null && isDirectory(parentKey);
   }
 
+  protected String initMultiPart(String key, MultipartUfsOptions options)
+      throws IOException {
+    throw new IOException("not support");
+  }
+
+  protected OutputStream uploadPart(String key, String uploadId, int partNum, ByteBuffer b,
+                                    MultipartUfsOptions options)
+      throws IOException {
+    throw new IOException("not support");
+  }
+
+  protected String uploadPartWithStream(String key, String uploadId, int partNum, long fileSize,
+                                        InputStream stream, MultipartUfsOptions options)
+      throws IOException {
+    throw new IOException("not support");
+  }
+
+  protected String completeMultiPart(String key, String uploadId, List<Pair<Integer, String>> etags,
+                                     MultipartUfsOptions options)
+      throws IOException {
+    throw new IOException("not support");
+  }
+
+  protected List<PartSummaryInfo> listParts(String key, String uploadId,
+                                            MultipartUfsOptions options)
+      throws IOException {
+    throw new IOException("not support");
+  }
+
+  public ListMultipartUploadResult listMultipartUploads(ListMultiPartOptions options)
```

Comment:
can you add a comment to explain what this method is for?

Reply:
for getting all MPU tasks in UFS. we can show it to the northbound S3 API